spark-connect/server/src/main/scala-spark-master/delta/connect/DeltaRelationPlugin.scala

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright (2024) The Delta Lake Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql.connect.delta
```

Comment:
We put the package name to be like this on purpose so that we can access some Spark Connect functions, which have package names with prefix `org.apache.spark.sql.connect` in Apache Spark that has limited private visibility, for example, this is needed to that `SparkEnv.get.conf.get(Connect.CONNECT_GRPC_MARSHALLER_RECURSION_LIMIT)` can work properly.